.gitignore

```diff
@@ -1,2 +1,2 @@
 *.pyc
-results
```

Comment:
Possibly, but with pytest-selenium and pytest-html this is no longer the default path. Also, the report is a single file so `results.html` is just as likely.